app/models/admin_menu.rb

```diff
@@ -0,0 +1,76 @@
+class AdminMenu
+  # On second level navigation with more children, we reference the default tabs controller. i.e look at developer_tools
+  # rubocop:disable Metrics/BlockLength
+  ITEMS = Menu.define do
+    scope :people, [
+      item(name: "people", controller: "users"),
+    ]
+
+    scope :content_manager, [
+      item(name: "posts", controller: "articles"),
+      item(name: "badges", children: [
+             item(name: "badge library", controller: "badges"),
+             item(name: "badge achievements"),
```

Comment:
this might be off topic, but is repeating the word "badge" and similar in sub-sections necessary?

I wonder if

![Screenshot_2021-03-05 Badges](https://user-images.githubusercontent.com/146201/110117376-0cacb900-7db9-11eb-8b58-58354c003e56.png)

is better or worse than

![Screenshot_2021-03-05 Badges(1)](https://user-images.githubusercontent.com/146201/110117402-16ceb780-7db9-11eb-98b3-968ad25b797c.png)

Reply:
This is a good point, I'm going to change it. cc @lisasy

Reply:
✅